low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -65,42 +64,42 @@ class ManifestSupportBundle(object):
 
 
     def make_manifest_support_bundle_tarfile(self, parser, tmpFile):
+        localTempPath = "/tmp/support_bundle/"
         configure_path = parser[2]
         if configure_path:
             if os.path.exists(configure_path):
-                self.localTempPath = configure_path+"manifest/"
+                localTempPath = configure_path+"manifest/"
             else:
                 msg = "Given path %s doesn't exist" %(configure_path)
                 tmpFile.close()
                 os.remove(onGoingRequestPath)
                 raise Exception(msg)
-        if not os.path.exists(self.localTempPath):
-            os.makedirs(self.localTempPath)
+        if not os.path.exists(localTempPath):
+            os.makedirs(localTempPath)
         try:
-            sspl_state = ""
             try:
                 lr_serial_number = open(self.lr_serial_number_path).readline().rstrip()
             except (FileNotFoundError, OSError) as e:
                 print(f"Unable to get lr serial number from the given path : {self.lr_serial_number_path}")
                 lr_serial_number = "NA"
-
             try:
-                sspl_state=open(f"/var/{PRODUCT_FAMILY}/sspl/data/state.txt").readline().rstrip().split("=")
+                self.support_common_data = {
+                    "cluster_id": self.cluster_id,
+                    "site_id": self.site_id,
+                    "rack_id": self.rack_id,
+                    "node_id": self.node_id
+                }
+                self.capture_node_data()
                 # Capturing enclosure data only when sspl state is active
-                sspl_state = sspl_state[1] if len(sspl_state)==2 else ''
-                if sspl_state == 'active':
-                    subprocess.call([f"{RESOURCE_HEALTH_VIEW}", "-n", "-e", "-s"], shell=False)
-                else:
-                    print("SSPL state is not active, So enclosure data are ignored to collecte.")
-                    subprocess.call([f"{RESOURCE_HEALTH_VIEW}", "-n", "-s"], shell=False)
-            except (FileNotFoundError, OSError) as e:
-                print(f"Failed to open the SSPL 'state.txt' file with an error '{e}',\
-                    Can't determine the SSPL state, So enclosure data also being collected.")
-                sspl_state == 'active'
-                subprocess.call([f"{RESOURCE_HEALTH_VIEW}", "-n", "-e", "-s"], shell=False)
+                if self.sspl_state == 'active':
+                    if self.storage_type == self.STORAGE_TYPE_RBOD:
+                        self.encl_manifest = f"{DATA_PATH}encl_manifest.json"
```

Comment:
> Add files in a specific folder(create if needed) for manifest, health, as applicable.

Comment not addressed
Create a folder undr DATA_PATH & use it.
like /var/cortx/sspl/data/manifest/

Reply:
Addressed

Reply:
Sorry, please use /var/cortx/sspl/data/supportbundle folder for tmp files for both support bundle & manifest

Reply:
Done.